### ISSUE TITLE
commends are not update in profile

### ISSUE THREAD

**Issue by Sahandfrz** [ValveSoftware/Dota2-Gameplay#12034]
### Description

Hello, I have a problem that has been around for about 2-3 weeks now, and my problem has not been solved, and no matter how much I searched on Google, I did not get anywhere.
My problem is that as far as I understand, the numbers of Commands and Match MVP variables do not change, even though I am much more than these numbers and these numbers are not updated, what should I do to solve the problem?
this problem is so annoying.

**Comment by shaileshgarg** [comment]
https://github.com/ValveSoftware/Dota2-Gameplay/issues/11278

**Comment by ligaapl** [comment]
Yes. I have 2172 commends a month ago and commends still not counted after profile update!
